test/end-to-end/paths/probate/singleExecutorsPath.js

```diff
@@ -22,7 +23,10 @@ After(() => {
     TestConfigurator.getAfter();
 });
 
-Scenario(TestConfigurator.idamInUseText('Single Executor Journey with sign out/in and survey link'), (I) => {
+Scenario(TestConfigurator.idamInUseText('Single Executor Journey with sign out/in and survey link'), async (I) => {
+
+    const useNewDeathCertFlow = await TestConfigurator.checkFeatureToggle(config.featureToggles.ft_new_deathcert_flow);
+    console.log('TESTING MEGAN!!!' + useNewDeathCertFlow);
```

Comment:
Lol maybe remove this console log or change the message

Reply:
It's just in for now so I can test it on the PR hahah